### PR TITLE
Masota/modify createconfigpage

### PR DESCRIPTION
**Pull request by NT-D**
As described #153 , some scheme and sdk stay old. I changed these with current version scheme.

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/createconfigpage.md](https://github.com/NT-D/Microsoft-teams-docs/blob/masota/ModifyCreateconfigpage/teams/createconfigpage.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C21%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201705210604434521-154%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/createconfigpage.md](https://github.com/NT-D/Microsoft-teams-docs/blob/masota/ModifyCreateconfigpage/teams/createconfigpage.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C22%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201705221616264330-154%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by SheriEllisMSFT** [comment]
@dfederm David: could you please review this PR? Thanks!

File: teams/createconfigpage.md [hunk 1]
```diff
@@ -42,15 +42,15 @@ microsoftTeams.settings.registerOnSaveHandler(function(saveEvent){
        microsoftTeams.settings.setSettings({
          entityId: "bing",
          contentUrl: "https://www.bing.com/maps/embed",
-         suggestedDisplayName: "Bing Map",
+         suggestedTabName: "Bing Map",
```
Comment (dfederm) [inline teams/createconfigpage.md]: v1.0 uses suggestedDisplayName

File: teams/createconfigpage.md [hunk 2]
```diff
@@ -42,15 +42,15 @@ microsoftTeams.settings.registerOnSaveHandler(function(saveEvent){
...
          websiteUrl: "https://www.bing.com/maps",
          removeUrl: "https://teams-get-started-sample.azurewebsites.net/tabremove.html",
       });
     } else {
        microsoftTeams.settings.setSettings({
          entityId: "google",
          contentUrl: "https://www.google.com/maps/embed",
-         suggestedDisplayName: "Google Map",
+         suggestedTabName: "Google Map",
```
Comment (dfederm) [inline teams/createconfigpage.md]: Same

File: teams/createconfigpage.md (hunk 1 above)
Comment (SheriEllisMSFT) [inline teams/createconfigpage.md]: @dfederm - but the first change configUrl to configurationUrl looks accurate to me from the javascript library, right?

@NT-D - please remove suggestedTabName, it has not changed in v1.0. Thanks!

Comment (dfederm) [inline teams/createconfigpage.md]: Yes, `configurationUrl` is correct

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/createconfigpage.md](https://github.com/NT-D/Microsoft-teams-docs/blob/masota/ModifyCreateconfigpage/teams/createconfigpage.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C23%5Ceddc9719-9a47-bd10-0ebb-4ef3276f7a0e%5CPullRequest%5C201705232203099132-154%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: teams/createconfigpage.md (hunk 1 above)
Comment (NT-D) [inline teams/createconfigpage.md]: @dfederm , @SheriEllisMSFT , thanks for reviewing. I removed suggestedTabName!

File: teams/createconfigpage.md (hunk 2 above)
Comment (NT-D) [inline teams/createconfigpage.md]: Thanks for your advice. Resolved it too.

**Pull request merged**